CHANGELOG.md

```diff
@@ -52,6 +52,14 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - `Supervisor` service now can have initial configuration and implements
   `Configure` interface. (#1587)
 
+### Internal Improvements
```

Comment:
I believe this is a breaking change, since import paths for *proof indexes were changed.

Reply:
Not sure if it really breaking, because external APIs remain unchanged.

Reply:
Import paths for *proof indexes only changed in MerkleDB.

Reply:
Ah, sure.